libraries/SITL/examples/JSON/C++/SocketExample.h

```diff
@@ -68,4 +76,9 @@ class SocketExample {
     int fd = -1;
 
     void make_sockaddr(const char *address, uint16_t port, struct sockaddr_in &sockaddr);
+
+#ifdef _WIN32
+    bool socket_init();
+#endif // _WIN32
+
```

Comment:
`socket_init` is used in `SocketExample::SocketExample(bool _datagram, int _fd)` without the include guard, so these ones either need removing or the guards need to be in place in the cpp.